types/bootstrap/js/dist/base-component.d.ts

```diff
@@ -3,6 +3,7 @@
  * each plugin exposes a getInstance method. In order to retrieve it directly from an element, do this: bootstrap.Popover.getInstance(myPopoverEl)
  */
 export type GetInstanceFactory<T> = (element: string | Element) => T | null;
+export type GetOrCreateInstanceFactory<T> = (element: string | Element) => T;
```

Comment:
The method has two patameters. Please see bootstrap repository and add the second parameter. 

static getOrCreateInstance(element, config = {}) {
    return this.getInstance(element) || new this(element, typeof config === 'object' ? config : null)
  }

Reply:
see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55213/commits/5e19743e2b6586383a74b26ee6debcfe1c7a12e2